phidgets_high_speed_encoder/src/high_speed_encoder_ros_i.cpp

```diff
@@ -74,6 +74,45 @@ HighSpeedEncoderRosI::HighSpeedEncoderRosI(ros::NodeHandle nh,
     {
         publish_rate_ = 0;
     }
+    if (!nh_private.getParam("data_interval", data_interval_))
+    {
+        data_interval_ = 0;
```

Comment:
So this is a deviation from the other phidgets, where we always explicitly set a default value. But actually I like your version better (leaving the device default). So no need to change anything here, just a remark.

Reply:
Oh, one more thing: This parameter has to be renamed to `data_interval_ms`, because that's what all the other phidgets use.